### PR TITLE
Leapp Preupgrade report - Add upgrade button

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: webpack/components/PreupgradeReports/PreupgradeReports.js [hunk 1]
```diff
@@ -64,6 +65,12 @@ const PreupgradeReports = ({
               csrfToken={csrfToken}
               preupgradeReports={preupgradeReports}
             />
+            <UpgradeAllButton
+              postUrl={newJobInvocationUrl}
+              disabled={false}
```
Comment (xprazak2) [inline webpack/components/PreupgradeReports/PreupgradeReports.js:70]: nit: You can remove the `disabled` prop if button will be always enabled.

File: webpack/components/PreupgradeReports/components/UpgradeAllButton.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import PropTypes from 'prop-types';
+import { Button } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import { idsForInvocation } from '../PreupgradeReportsHelpers';
+
+const UpgradeAllButton = ({
+  preupgradeReports,
+  postUrl,
+  disabled,
```
Comment (xprazak2) [inline webpack/components/PreupgradeReports/components/UpgradeAllButton.js:11]: Can be removed.

File: webpack/components/PreupgradeReports/components/UpgradeAllButton.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  csrfToken,
+}) => {
+  const { hostIds } = idsForInvocation(preupgradeReports);
+
+  return (
+    <form action={postUrl} method="post">
+      <Button type="submit" disabled={disabled}>
```
Comment (xprazak2) [inline webpack/components/PreupgradeReports/components/UpgradeAllButton.js:18]: `disabled` can be removed here as well.

File: webpack/components/PreupgradeReports/components/UpgradeAllButton.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <input type="hidden" name="host_ids[]" key={hostId} value={hostId} />
+      ))}
+    </form>
+  );
+};
+
+UpgradeAllButton.propTypes = {
+  preupgradeReports: PropTypes.array.isRequired,
+  postUrl: PropTypes.string.isRequired,
+  disabled: PropTypes.bool.isRequired,
```
Comment (xprazak2) [inline webpack/components/PreupgradeReports/components/UpgradeAllButton.js:33]: Same as above.

File: webpack/components/PreupgradeReports/components/UpgradeAllButton.test.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+import UpgradeAllButton from './UpgradeAllButton';
+
+import { preupgradeReports } from '../__tests__/PreupgradeReports.fixtures';
+
+const fixtures = {
+  'should render': {
+    preupgradeReports,
+    postUrl: '/job_invocations/new',
+    disabled: false,
```
Comment (xprazak2) [inline webpack/components/PreupgradeReports/components/UpgradeAllButton.test.js:11]: Can be removed in tests as well.

**Comment by xprazak2** [comment]
Looks good!

**Comment by ares** [comment]
And works fine too, just finished testing. I think this can be merged if there's no other comments.

**Comment by adamruzicka** [comment]
Thank you @stejskalleos , @xprazak2 & @ares !

**Pull request merged**